detox/src/devices/drivers/android/emulator/EmulatorDriver.js

```diff
@@ -63,6 +58,10 @@ class EmulatorDriver extends AndroidDriver {
     return adbName;
   }
 
+  async doAllocateDevice(deviceQuery) {
+    return this.freeDeviceFinder.findFreeDevice(deviceQuery);
```

Comment:
@DmytroKost missing an `await` here
```suggestion
    return await this.freeDeviceFinder.findFreeDevice(deviceQuery);
```

Reply:
@d4vidi, why do you want an explicit `await`  there?